.github/workflows/cypress-ci.yml

```
@@ -1,6 +1,6 @@
-name: Portal CI
+name: Cypress CI
 
-on: [push, pull_request]
+on: [pull_request]
```

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```

We only want ci to run once for PRs and their updates, but we also want ci to run on pushes to main? With `pull_request` alone CI wouldn't run once PRs are merged to main. This applies to the other actions as well.